src/css/quasar.variables.scss

```diff
@@ -12,13 +12,30 @@
 // to match your app's branding.
 // Tip: Use the "Theme Builder" on Quasar's documentation website.
 
-$primary   : #1976D2;
-$secondary : #26A69A;
-$accent    : #9C27B0;
+$primary: #010101;
+$secondary: #ffffff;
+$accent: #9c27b0;
 
-$dark      : #1D1D1D;
+$dark: #1d1d1d;
 
-$positive  : #21BA45;
-$negative  : #C10015;
-$info      : #31CCEC;
-$warning   : #F2C037;
+$positive: #21ba45;
+$negative: #c10015;
+$info: #31ccec;
+$warning: #f2c037;
+
+// Custom colors
+$white-flight: #f2f2f2;
+$gray-dim: #707070;
+$gray-chinese-silver: #ccc7c7;
```

Comment:
What about using this
https://github.com/dradisavljevic/ColorfulScraper

To get the names and values in a JSON?

It's weird to see a wild name appearing each time a new color is added hehe

Reply:
Also would do an easy open-source project that other would find useful

Reply:
I think we need to define the design system, but it's difficult since the design doesn't have color variables saved :( In fact, I still can't see the color / text / adobe xd development tools on the design

![image](https://user-images.githubusercontent.com/66505715/170686639-cb662ce4-2a1f-439d-aad4-231e7319a52c.png)

But I think mixing JS with CSS will complicate the development since Quasar is not made to work with JavaScript variables.

Reply:
Unless a library like TailwindCSS was installed since it converts the JSON into SCSS in a syntax equal to the one used by Quasar for its variables

Reply:
I've found this library that could work: https://github.com/pmowrer/node-sass-json-importer @Israel-Laguan what do you think?

Reply:
Ah, what about using this in quasar:

![image](https://user-images.githubusercontent.com/36519478/170739883-683639bd-80be-4dfd-9cda-db1f0d6b8b43.png)

https://quasar.dev/style/color-palette#util-setcssvar

Reply:
Good solution, I'll try to apply it

Reply:
But I think it isn't needed to add the 1690 colors that have the encycolorpedia, just the required colors by the designer.

I could create a new PR to add this feature of use Quasar colors

Reply:
I've created a new Issue to handle the colors using this way that you mentions @Israel-Laguan  https://github.com/Azordev/did-app/issues/33

Reply:
I was checking setCssVar just works with the brand colors (primary, secondary, accent, dark, positive, negative, info, warning)
![image](https://user-images.githubusercontent.com/66505715/170890371-c33edf64-a887-4b50-b12a-168dab4a3a2c.png)

Reply:
@Israel-Laguan I was checking and Quasar recommends creating the colors using this way (current way)

https://quasar.dev/style/color-palette#adding-your-own-colors

![image](https://user-images.githubusercontent.com/66505715/170890620-4e0c41bc-9f60-4827-a6aa-f7ce4ec1d8cf.png)

and warns that setCssVar couldn't work with custom colors, if not just with brand colors (primary, secondary, etc...)

![image](https://user-images.githubusercontent.com/66505715/170890643-3cbec9aa-73d3-4a6d-ac0d-047222ac687c.png)

But I could try to use setCssVar.

I will add a discussion about this to avoid block this PR

Reply:
We could continue the conversation about how to handle the colors, here: https://github.com/Azordev/did-app/discussions/34